docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.5"
 
 services:
   cardano-node:
-    image: inputoutput/cardano-node:1.24.2
+    image: inputoutput/cardano-node:1.25.1
```

Comment:
Since cardano-node 1.25.1 is not yet officially released, there is no 1.25.1 tag on docker hub yet... :shrug:

Reply:
Ah!

Reply:
We're living on the edge.

Reply:
https://github.com/input-output-hk/cardano-node/releases/tag/1.25.1

:thinking:  ?

Reply:
Yes it's tagged, but no docker image for 1.25.1 until it's released. 

https://hub.docker.com/r/inputoutput/cardano-node/tags?page=1&ordering=last_updated